doc/introduction.rst

```diff
@@ -6,42 +6,56 @@ Introduction
+See :ref:`register-file` for more details.
+The core occupies an area of roughly 18.9 kGE when using the latch-based register file and implementing the RV32IMC ISA, or 11.6 kGE when implementing the RV32EC ISA.
 
 FPGA Synthesis
 --------------
 
-FPGA synthesis is supported for Ibex when the flip-flop based register file is used. Since latches are not well supported on FPGAs, it is crucial to select the flip-flop based register file.
+FPGA synthesis is supported for Ibex when the flip-flop based register file is used.
+Since latches are not well supported on FPGAs, it is crucial to select the flip-flop based register file.
 
-Outline
--------
+Contents
+--------
+
+:ref:`core-integration` provides the instantiation template and gives descriptions of the design parameters as well as the input and output ports.
+The instruction and data interfaces of Ibex are explained in :ref:`instruction-fetch` and :ref:`load-store-unit`, respectively.
+The two register-file flavors are described in :ref:`register-file`.
+The control and status registers are explained in :ref:`cs-registers`.
+:ref:`performance-counters` gives an overview of the performance monitors and event counters available in Ibex.
+:ref:`exceptions-interrupts` deals with the infrastructure for handling exceptions and interrupts,
+:ref:`debug-support` gives a brief overview on the debug infrastructure.
+For information regarding formal verification support, check out :ref:`rvfi`.
 
-This document summarizes all the functionality of the Ibex core in more detail. First, the instruction and data interfaces are explained in :ref:`instruction-fetch` and :ref:`load-store-unit`. :ref:`register-file` explains the register file. Control and status registers are explained in :ref:`cs-registers` and Chapter :ref:`performance-counters` gives an overview of all performance counters. Chapter :ref:`exceptions-interrupts` deals with exceptions and interrupts, and finally Chapter :ref:`debug-unit` summarizes the accessible debug registers.
 
 History
 -------
 
-Ibex development started in 2015 under the name "ZERO-RISCY" as part of the `PULP platform <http://pulp.ethz.ch/>`_ for
-energy-efficient computing.
-Much of the code was developed by simplifying the RV32 CPU core "RI5CY" to show how small a RISC-V CPU core could be.
-To make it even smaller, support for the ``E`` exension was added under the code name "MICRO-RISCY".
-Within PULP the core is used as the control core for PULP and PULPino.
+Ibex development started in 2015 under the name "Zero-riscy" as part of the `PULP platform <https://pulp-platform.org>`_ for energy-efficient computing.
+Much of the code was developed by simplifying the RV32 CPU core "RI5CY" to demonstrate how small a RISC-V CPU core could actually be.
+To make it even smaller, support for the "E" exension was added under the code name "Micro-riscy".
+In the PULP ecosystem, the core is used as the control core for PULP, PULPino and PULPissimo.
```

Comment:
As Davide said, linking the original paper from here would be nice (and much better than in the README). 
https://ieeexplore.ieee.org/document/8106976/ (Is there a free preprint available?)

Reply:
Unfortunately, there is no free preprint available. We usually did that. But maybe the conference did not allow it. But I will ask Davide.

Reply:
Looks like researchgate has one available. We don't need to link to it, people will find it if they are looking for it.

https://www.researchgate.net/profile/Francesco_Conti4/publication/321121782_Slow_and_steady_wins_the_race_A_comparison_of_ultra-low-power_RISC-V_cores_for_Internet-of-Things_applications/links/5a37aa3ea6fdccdd41fc99e1/Slow-and-steady-wins-the-race-A-comparison-of-ultra-low-power-RISC-V-cores-for-Internet-of-Things-applications.pdf